### PR TITLE
Fix18970

### LINKED ISSUE #18970: Add more assertions for actual value with method AssistQueryAndPlainInsertColumnsTokenGeneratorTest.assertGenerateSQLTokens()

### PR DESCRIPTION
**Pull request by huangdx0726**
Fixes #18970 .

Changes proposed in this pull request:
-
add more tests
-

### CODE REVIEW

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/AssistQueryAndPlainInsertColumnsTokenGeneratorTest.java [hunk 1]
```diff
@@ -67,7 +68,30 @@ public void assertGenerateSQLTokens() {
         tokenGenerator.setEncryptRule(mockEncryptRule());
         Collection<InsertColumnsToken> actual = tokenGenerator.generateSQLTokens(mockInsertStatementContext());
         assertThat(actual.size(), is(1));
-        // TODO add more assertions for actual value
+
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/AssistQueryAndPlainInsertColumnsTokenGeneratorTest.java:71]: Please remove unnecessary blank line

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/AssistQueryAndPlainInsertColumnsTokenGeneratorTest.java [hunk 2]
```diff
@@ -67,7 +68,30 @@ public void assertGenerateSQLTokens() {
...
+        insertStatementContext = mock(InsertStatementContext.class, RETURNS_DEEP_STUBS);
+        when(insertStatementContext.getSqlStatement().getTable().getTableName().getIdentifier().getValue()).thenReturn("foo_tbl");
+        actual = tokenGenerator.generateSQLTokens(insertStatementContext);
+        assertThat(actual.size(), is(0));
+
+        columnSegment = mock(ColumnSegment.class, RETURNS_DEEP_STUBS);
+        when(columnSegment.getIdentifier().getValue()).thenReturn("bar_col");
+        when(insertStatementContext.getSqlStatement().getColumns()).thenReturn(Collections.singleton(columnSegment));
+        actual = tokenGenerator.generateSQLTokens(insertStatementContext);
+        assertThat(actual.size(), is(0));
```
[the remark this answers is not in the document]
Comment (huangdx0726) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/AssistQueryAndPlainInsertColumnsTokenGeneratorTest.java:94]: I will fix it

**Comment by terrymanu** [comment]
@huangdx0726 Hi, any update?

**Comment by huangdx0726** [comment]
fixed&nbsp; by&nbsp; comment&nbsp;



---Original---
From: "Liang ***@***.***&gt;
Date: Wed, Jul 20, 2022 11:34 AM
To: ***@***.***&gt;;
Cc: "Da Xiang ***@***.******@***.***&gt;;
Subject: Re: [apache/shardingsphere] Fix18970 (PR #19027)




 
@huangdx0726 Hi, any update?

**Review by terrymanu — APPROVED** [review]

**Pull request merged**